java/Cherepnin_Oleh/src/com/zoolatech/lecture4/tasks/_5/TaskFive.java

```diff
@@ -0,0 +1,48 @@
+package com.zoolatech.lecture4.tasks._5;
+
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.math.RoundingMode;
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts two numbers and a symbol of an operation (as a character) and outputs the result
+ * of the selected operation. Possible characters for the operation: ‘+’, ‘-’, ‘*’, ‘/’, ‘%’. The program should
+ * work with decimal numbers of any length and precision.
+ */
+public class TaskFive {
+
+    private static Scanner scanner = new Scanner(System.in);
+    private static Calculator calculator = (v1, v2, op) -> {
```

Comment:
Same: try to keep your code simple - you could use a lambda if you had a need to pass this code somewhere else or dynamically change it with some other implementation.

However, here it's enough to have a single method that will perform calculations